labkey-api-sas/README.md

```diff
@@ -0,0 +1,21 @@
+# The LabKey Remote API Library for SAS
+
+Library of macros for use in accessing LabKey Server via SAS.
+
+For more information on the library, including setup instructions and
+examples, see our [SAS API](https://www.labkey.org/Documentation/wiki-page.view?name=sasAPI)
+documentation.
+
+If you have any questions about this library or the LabKey Server, 
+see our [Community Support Forums](https://www.labkey.org/home/Support/LabKey%20Support%20Forum/project-begin.view?).
```

Comment:
I'd remove these two lines. We don't really want to support this library on the community forums